Assets/Scripts/Interaction/Hand.cs

```diff
@@ -10,6 +10,9 @@ public class Hand : MonoBehaviour {
     float SIGIL_DISTANCE = 0.05F;
     float HIT_ARRAY_DISTANCE = 0.6F;
     float HIT_ARRAY_VERT_BIAS = 0.2F;
+    static GameObject currentProjectile = null;
```

Comment:
This static means we can only ever have one projectile. I don't think that is intended behaviour